frontend/src/scenes/insights/BreakdownFilter/BreakdownFilter.js

```diff
@@ -110,6 +110,7 @@ function Content({ breakdown, breakdown_type, onChange }) {
         >
             <TabPane tab="Property" key="property">
                 <PropertyFilter
+                    pageKey={`${breakdown}-${window.location.pathname}`}
```

Comment:
What did this addition fix?

Reply:
Resolved — realized this is not the same component as the others, although its name is identical